packages/renderer/src/lib/extensions/ExtensionDetailsReadme.spec.ts

```diff
@@ -19,19 +19,12 @@
 import '@testing-library/jest-dom/vitest';
 
 import { render, screen } from '@testing-library/svelte';
-import { tick } from 'svelte';
 import { beforeEach, expect, test, vi } from 'vitest';
 
 import ExtensionDetailsReadme from './ExtensionDetailsReadme.svelte';
 
 async function waitRender(customProperties: object): Promise<void> {
   render(ExtensionDetailsReadme, { ...customProperties });
-  await tick();
-  await tick();
-  await tick();
-  await tick();
-  await tick();
-  await tick();
```

Comment:
WaitFor is so much cleaner than `tick()` 👍

Reply:
I added a comment in the code guideline https://github.com/podman-desktop/podman-desktop/issues/9821#issuecomment-2567444651